website/vercel.json

```diff
@@ -1201,6 +1202,56 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/reference/appsmith-framework/reference/appsmith-framework/widget-actions/post-message",
+      "destination": "/reference/appsmith-framework/widget-actions/post-message",
+      "permanent": true
+    },
+    {
+      "source": "/build-apps/how-to-guides/reference/appsmith-framework/widget-actions/window-message-listener",
+      "destination": "/reference/appsmith-framework/widget-actions/window-message-listener",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/file-picker",
+      "destination": "/reference/widgets/filepicker",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/map",
+      "destination": "/reference/widgets/maps",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
+      "destination": "/connect-data/reference/querying-mongodb",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/radio-buttons",
```

Comment:
This is the wrong URL why do we need a redirect? This should be removed from the index.